docusaurus/docs/cloud/projects/overview.md

```diff
@@ -32,35 +34,33 @@ Each project card displays the following information:
+## Accessing a project's dashboard
 
-- links to the other available tabs for the project: *Overview*, [*Deploys*](/cloud/projects/deploys), [*Runtime Logs*](/cloud/projects/runtime-logs) and [*Settings*](/cloud/projects/settings)
-- the **Share** button to invite a new maintainer to collaborate on your project — and if the project is already shared: avatars of the maintainers (see [Collaboration](/cloud/projects/collaboration))
-- the **Trigger deploy** button to trigger a new deployment of your project
-- the **Visit app** button to access your application
-:::
+From the *Projects* page, click on any project card to access is dashboard. It displays the project and environment details and gives access to the deployment history and all available settings.
 
 <ThemedImage
   alt="Project overview"
   sources={{
-    light: '/img/assets/cloud/overview.png',
-    dark: '/img/assets/cloud/overview_DARK.png',
+    light: '/img/assets/cloud/overview2.png',
+    dark: '/img/assets/cloud/overview2_DARK.png',
   }}
 />
 
-From the *Overview* tab, you can:
-- view a recap of the main settings of your project, such as:
-  - the link to the source repository
-  - the name of the branch
-  - the name of the base directory
-  - the URL and link to the application
-- view your project's usage (see [Usage](/cloud/getting-started/usage-billing) for more information)
-- view your project's latest deploys (see [Deploys](/cloud/projects/deploys) for more information)
+From the dashboard's header of a chosen project, you can:
+- use the **Share** button to invite users to collaborate on the project (see [Collaboration](/cloud/projects/collaboration)) and see the icones of those who have already been invited <ScreenshotNumberReference number="1" />,
+- use the ![Settings icon](/img/assets/icons/settings.svg) button to access the settings of the project and its existing environments <ScreenshotNumberReference number="2" />,
+- choose which environment to visualise for the project <ScreenshotNumberReference number="3" />,
+- trigger a new deployment (see [Deployments management](/cloud/projects/deploys)) and visit your application <ScreenshotNumberReference number="4" />.
+
+Your project's dashboard also displays:
+- the *Deployments* and *Runtime logs* tabs, to see the deployments history (more details in [Deploy history and logs](/cloud/projects/deploys-history)) and the runtime logs of the project (see [dedicated documentation page](/cloud/projects/runtime-logs)) <ScreenshotNumberReference number="5" />
+- the project and environment details in a box on the right of the interface <ScreenshotNumberReference number="6" />, including:
+  - the number of API calls and database entries,
+  - the current usage for assets bandwidth and storage,
+  - the name of the branch and a **Manage** button to be redirect to the branch settings (see Modifying git repository & branch),
+  - the name of the base directory,
+  - the Strapi version number.
+
```

Comment:
Heads up, we will be adding a link to the environment's CMS site and a copy icon. I'll let you know when we start working on this